packages/amazonq/src/lsp/client.ts

```diff
@@ -32,14 +33,21 @@ import {
     getLogger,
     undefinedIfEmpty,
     getOptOutPreference,
+    isAmazonInternalOs,
+    fs,
 } from 'aws-core-vscode/shared'
 import { activate } from './chat/activation'
 import { AmazonQResourcePaths } from './lspInstaller'
 import { ConfigSection, isValidConfigSection, toAmazonQLSPLogLevel } from './config'
+import { chmodSync } from 'fs' // eslint-disable-line no-restricted-imports
```

Comment:
We should use the wrapper [here](https://github.com/aws/aws-toolkit-vscode/blob/457efa4efff9cdf0d2c82d165063bd6b4c64cd6b/packages/core/src/shared/fs/fs.ts#L347). It doesn't do anything on browsers, but has the same behavior in a node process.